pkg/version/version.go

```diff
@@ -1,13 +1,51 @@
 package version
 
+import (
+	"encoding/json"
+
+	"github.com/aws/eks-anywhere/pkg/files"
+	"github.com/aws/eks-anywhere/pkg/manifests/releases"
+)
+
 var gitVersion string
 
 type Info struct {
-	GitVersion string
+	GitVersion        string
+	BundleManifestURL string
 }
 
 func Get() Info {
 	return Info{
 		GitVersion: gitVersion,
 	}
 }
+
+// GetFullVersionInfo returns the complete version information for the
+// EKS Anywhere, including Git version and bundle manifest URL
+// associated with this release.
+func GetFullVersionInfo() (Info, error) {
+	reader := files.NewReader(files.WithEKSAUserAgent("cli", gitVersion))
+	bundleManifestURL, err := releases.GetBundleManifestURLForVersion(reader, gitVersion)
+	if err != nil {
+		return Info{GitVersion: gitVersion}, err
+	}
+
+	return Info{
+		GitVersion:        gitVersion,
+		BundleManifestURL: bundleManifestURL,
+	}, nil
+}
+
+// GetFullVersionInfoJSON returns the complete version information for the
+// EKS Anywhere, including Git version and bundle manifest URL
+// associated with this release, formatted as a JSON string.
+func GetFullVersionInfoJSON() string {
+	versionInfo, err := GetFullVersionInfo()
+	if err != nil {
+		return ""
```

Comment:
if `GetFullVersionInfo` returns an error it seems like it will be impossible to troubleshoot, no?

Reply:
I can make the output similar to `GetFullVersionInfoJSON`'s error case, where it gets the EKS-A Git version atleast and then errors out.

Reply:
Fixed this by surfacing all errors. In case of an error, the Git Version will be printed out, and will be followed by the error log.